processing/src/main/java/org/apache/carbondata/processing/store/writer/AbstractFactDataWriter.java

```diff
@@ -252,6 +252,9 @@ private static long getMaxOfBlockAndFileSize(long blockSize, long fileSize) {
     if (remainder > 0) {
       maxSize = maxSize + HDFS_CHECKSUM_LENGTH - remainder;
     }
+    LOGGER.info("The configured block size is " + blockSize + " byte, " +
```

Comment:
Suggest to have the `blockSize` convert to a proper number before logging it, otherwise it is hard to check this value by human